.infra/Freyfile.hcl

```diff
@@ -167,7 +182,6 @@ setup {
     }
     roles {
       role = "{{{init.paths.roles_dir}}}/fqdn/1.2.1"
-      fqdn = "{{lookup('env', 'FREY_DOMAIN')}}"
```

Comment:
I should not break since it's defined [here](https://github.com/tus/tusd/blob/master/.infra/group_vars/all/_frey.yml#L25) now, but I can give it another test run if you want to :-)